### PR TITLE
Fix action info table not rendering correctly in docs

### PR DESCRIPTION
**Pull request by revolter**
Before:

screen shot 2018-04-09 at 23 16 18

After:

screen shot 2018-04-09 at 23 16 26

I didn't find a way to make the header cell span 2 columns.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/commit_github_file.rb [hunk 1]
```diff
@@ -153,7 +153,7 @@ def self.return_value
         [
           "A hash containing all relevant information for this commit",
           "Access things like 'html_url', 'sha', 'message'"
-        ].join("\n")
+        ].join("<br>")
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/commit_github_file.rb]: This needs to stay a `\n` for terminal output 🙃 

screen shot 2018-04-10 at 5 34 17 am

File: fastlane/lib/fastlane/actions/set_github_release.rb [hunk 2]
```diff
@@ -236,7 +236,7 @@ def self.return_value
         [
           "A hash containing all relevant information of this release",
           "Access things like 'html_url', 'tag_name', 'name', 'body'"
-        ].join("\n")
+        ].join("<br>")
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/actions/set_github_release.rb]: Same here 😇

**Review by joshdholtz — CHANGES_REQUESTED** [review]

File: fastlane/lib/fastlane/actions/commit_github_file.rb (hunk 1 above)
Comment (revolter) [reply fastlane/lib/fastlane/actions/commit_github_file.rb]: Damn :( Thanks for catching this! This means I need to `gsub` `\n` with `<br>` in the `erb`, right? I'll test it tonight.

**Review by joshdholtz — APPROVED** [review]
💯 💪

**Pull request merged**